R/db-odbc-mssql.R

```diff
@@ -100,3 +94,14 @@
   )
   DBI::dbExecute(con, sql)
 }
+
+mssql_is_null <- function(x, sql_context){
+  if ( !is.null(sql_context$mutate) | !is.null(sql_context$arrange) ) {
```

Comment:
I think it would be better to have parameters `x` and `op`.

The indenting also looks a bit off